DynamicQueryBuilder.UnitTests/DynamicQueryAttributeTests/OnActionExecutingTests.cs

```diff
@@ -53,19 +49,32 @@ public void ShouldSetGivenCtorValues()
             var parameterlessInstance = new DynamicQueryAttribute();
             var parameteredInstance = new DynamicQueryAttribute(200, false, PaginationBehaviour.Throw);
 
-            Assert.Equal(parameterlessInstance._maxCountSize, _defaultMaxCountSize);
-            Assert.Equal(parameterlessInstance._includeDataSetCountToPagination, _defaultIncludeDataSetCount);
-            Assert.Equal(parameterlessInstance._exceededPaginationCountBehaviour, _defaultPaginationBehaviour);
+            Assert.Equal(parameterlessInstance._maxCountSize, null);
+            Assert.Equal(parameterlessInstance._includeDataSetCountToPagination, null);
+            Assert.Equal(parameterlessInstance._exceededPaginationCountBehaviour, null);
 
             Assert.Equal(parameteredInstance._maxCountSize, changedMaxCountSize);
             Assert.Equal(parameteredInstance._includeDataSetCountToPagination, changedIncludedataSetCount);
             Assert.Equal(parameteredInstance._exceededPaginationCountBehaviour, changedPaginationBehaviour);
         }
 
+        [Fact]
+        public void ShouldNotSetCTorValuesWhenApplyGlobalFilter()
+        {
+            var defaultInstance = new DynamicQueryAttribute();
+            string queryExceedingMaxCount = DYNAMIC_QUERY_STRING.Replace("count=10", $"count={DynamicQueryAttributeGlobalConfig.MaxCountSize + 1}");
```

Comment:
As far as i understand we are missing integration tests here. Can we please add those as well ?